w3m

```diff
@@ -0,0 +1,116 @@
+<h1 id="cldr-remove-number-of-decimals-in-shop-parameter-general">[CLDR] Remove Number of decimals in Shop Parameter &gt; General</h1>
```

Comment:
I think this file doesn't belong here :)

Reply:
actually there was 2 wrong files 🤦‍♂   I fixed it thanx !